src/engraving/libmscore/image.cpp

```diff
@@ -77,7 +79,7 @@ Image::Image(const Image& img)
     if (imageType == ImageType::RASTER) {
         rasterDoc = img.rasterDoc ? new QImage(*img.rasterDoc) : 0;
     } else if (imageType == ImageType::SVG) {
-        svgDoc = img.svgDoc ? new QSvgRenderer(_storeItem->buffer()) : 0;
+        svgDoc = img.svgDoc ? new SvgRenderer(_storeItem->buffer()) : 0;
```

Comment:
so maybe we just need to add "setBuffer" method there, so we don't need to recreate it in "layout()" method every time

Reply:
I think yes, needs refactoring for this class (by way, there is a memory leak too).
But I would not like to mix different tasks in one PR